src/netspresso_trainer/models/op/custom.py

```diff
@@ -848,3 +848,32 @@ def __init__(self, dim: int, init_values: float = 1e-5, inplace: bool = False):
     def forward(self, x):
         gamma = self.gamma.view(1, -1, 1, 1)
         return x.mul_(gamma) if self.inplace else x * gamma
+
+
+class ELAN(nn.Module):
+    """
+    basic ELAN structure.
+    """
```

Comment:
Can you add paper reference where this module idea came from?
And if this code is copied version and not self-implemented from paper, please add reference of original codebase.

Reply:
Thank you for reviewing the implementation. 
I would upload the url for the reference with additional commit! 👍
ref: https://github.com/WongKinYiu/YOLO/blob/main/yolo/model/module.py